cmd/prometheus/main.go

```diff
@@ -1737,3 +1737,33 @@ type discoveryManager interface {
 	Run() error
 	SyncCh() <-chan map[string][]*targetgroup.Group
 }
+
+// TODO(bwplotka): Add unit test.
+type rwProtoMsgFlagParser struct {
+	msgs *[]config.RemoteWriteProtoMsg
+}
+
+func rwProtoMsgFlagValue(msgs *[]config.RemoteWriteProtoMsg) kingpin.Value {
+	return &rwProtoMsgFlagParser{msgs: msgs}
+}
+
+func (p *rwProtoMsgFlagParser) IsCumulative() bool {
+	return true
+}
```

Comment:
it's not clear what this is here for, is it part of `kingpin.Value`?

Reply:
correct, will address in the next one 🤗